Src/VsSpecific/VsSpecific/CSharpScriptExecutor.cs

```diff
@@ -0,0 +1,115 @@
+﻿#if VS_SPECIFIC_2017 || VS_SPECIFIC_2019
+
+using Microsoft.CodeAnalysis.CSharp.Scripting;
+using Microsoft.CodeAnalysis.Scripting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using Vim.Interpreter;
+
+namespace Vim.VisualStudio.Specific
+{
+    internal sealed class CSharpScriptExecutor : ICSharpScriptExecutor
+    {
+        private const string ScriptFolder = "vsvimscripts";
+        private Dictionary<string, Script<object>> _scripts = new Dictionary<string, Script<object>>(StringComparer.OrdinalIgnoreCase);
+        private ScriptOptions _scriptOptions = null;
+
+        private void Execute(IVim vim, CallInfo callInfo, bool createEachTime)
+        {
+            try
+            {
+                Script<object> script;
+                if (!TryGetScript(vim, callInfo.Name, createEachTime, out script))
+                    return;
+
+                var globals = new CSharpScriptGlobals(callInfo, vim);
+                script.RunAsync(globals).Wait();
+            }
+            catch (CompilationErrorException ex)
+            {
+                if (_scripts.ContainsKey(callInfo.Name))
+                    _scripts.Remove(callInfo.Name);
+
+                vim.ActiveStatusUtil.OnError(string.Join(Environment.NewLine, ex.Diagnostics));
+            }
+            catch (Exception ex)
+            {
+                vim.ActiveStatusUtil.OnError(ex.Message);
+            }
+        }
+
+        private ScriptOptions GetScriptOptions(string scriptPath)

```

Comment:
Believe this could be `static`

Reply:
I would like to fix this.
But I had never minded that private methods should be static.
Why should I do this?
Is it because memory can be saved by this change?